scc/committee_test.go

```diff
@@ -79,6 +79,29 @@ func TestCommittee_GetMemberId_ReturnsNotFoundIfNotPresent(t *testing.T) {
 	require.False(t, found)
 }
 
+func TestCommittee_GetTotalVotingPower_ReturnsCorrectTotal(t *testing.T) {
+	total, overflow := Committee{}.GetTotalVotingPower()
```

Comment:
should this one and the test below use `NewCommitteeForTests` ?

Reply:
All of the tests in this file are creating the `Committee{}` directly, and as these are the unit tests specifically for this type, I think it is OK for them to do it like this. It is also less opaque, since no side effects can be hidden in the function call.